src/commands/init/generateTemplate/index.js

```diff
@@ -54,7 +53,7 @@ export default function generateTemplate(name, src, dest, done) {
             }
             log.success('Project created');
             done();
-            logMessage(opts.completionMessage, data);
+            logMessage(opts.completionMessage, { ...data, ...metalsmith.metadata() });
```

Comment:
I have now looked into this some more I have found one bug/incorrect behavior, and one regression from #171.

1. I think we should actually mutate `metalsmith.metadata()`, something that seems like a somewhat normal pattern and it would make it possible to read `inPlace` and `destDirName` in the questions as well. Additionally `noEscape` is ignored after #171, and should actually me respected always and not just in the completion message.

I think we should either manage this in this PR or close this and open a new, that this does not work should be seen as a bug and not a feature.